apps-rendering/config/.eslintrc.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
 		project: 'tsconfig.json',
 	},
 	rules: {
+		'@typescript-eslint/no-unsafe-call': 'off',
```

Comment:
Would be nice to find a way to not have to do this if possible!

Reply:
Thanks Olly. Jamie's suggestion to update @types/serve-static worked so I deleted the `'@typescript-eslint/no-unsafe-call': 'off',` rule